src/reference/config/writing-platform-configs.rst

```diff
@@ -0,0 +1,207 @@
+
+.. _AdminGuide.PlatformConfigs:
+
+Writing Platform configurations
+===============================
+
+.. versionadded:: 8.0.0
+
+.. seealso::
+
+   - `The original platforms proposal on Cylc Admin <https://github.com/cylc/cylc-admin/blob/master/docs/proposal-platforms.md>`_.
```

Comment:
```suggestion
```

I'm not sure this is useful, Cylc will continue to drift from the propsal which could cause confusion.